featureFlags/README.md

```diff
@@ -0,0 +1,149 @@
+# FeatureFlags
+
+# FeatureFlag object
+
+```
+{
+    'name': string,
+    'id': string,
+    'created_at': timestamp,
+    'updated_at': timestamp,
+    'config': {'enabled': boolean},
+    'owner': string,
+    'launched_at': timestamp
+}
+```
+
+## **Requests**
+
+|                 Route                 |             Description                        |
+|:-------------------------------------:|:----------------------------------------------:|
+|  [GET /featureFlags](#get-featureFlags)        |   Returns all feature flags in system |
+|  [POST /featureFlags](#post-featureFlags)      |   Creates a feature flag              |
+|  [PATCH /featureFlags/:id](#patch-featureFlagsId)   | Updates data of the feature flag |
+|  [DELETE /featureFlags/:id](#delete-featureFlagsId) |   Deletes the feature flag       |
+
+
+
+## **GET /featureFlags**
+
+ Returns all feature flags in the system 
+
+- **Params**
+   None
+- **Query**
+   None
+- **Body**
+   None
+- **Headers**
+   None
+- **Cookie**
+   None    
+- **Success Response:**
+- **Code:** 200
+  -**Content:**
+
+```
+{
+  message: 'FeatureFlags returned successfully!'
+  featureFlags: [
+           {<featureFlag_object>},
+           {<featureFlag_object}
+         ]
+}
+```
+
+- **Error Response:**
+  - **Code:** 500
+    - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+
+
+## **POST /featureFlags** 
+
+Creates a feature flag 
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Body** `
```

Comment:
mention that these are required fields. And if a required field is not present, we might give a different HTTP response with status code 400. (i.e missing required fields). You can also add this in the possible response codes for this route.